src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx

```diff
@@ -358,7 +362,11 @@ export function RepeatingGroupTable({
                         key={`edit-${index}`}
                       >
                         <IconButton
-                          className={classes.tableEditButton}
+                          className={`${classes.tableEditButton} ${
+                            editIndex === index
+                              ? classes.editButtonActivated
+                              : ''
+                          }`}
```

Comment:
This would look a bit nicer and easier to read by using `classnames` package


```tsx
// add import
import cn from 'classnames';
```

```suggestion
                          className={cn(classes.tableEditButton, {
                            [classes.editButtonActivated]: editIndex === index,
                          })}
                          
```

Same for the AltinnMobileTableItem file